tests/Collection/CrudSpecFunctionalTest.php

```diff
@@ -48,8 +48,7 @@ public function setUp(): void
     {
         parent::setUp();
 
-        $this->expectedCollection = new Collection($this->manager, $this->getDatabaseName(), $this->getCollectionName() . '.expected');
-        $this->expectedCollection->drop();
+        $this->expectedCollection = $this->dropCollection($this->getDatabaseName(), $this->getCollectionName() . '.expected');
```

Comment:
Noted that `$this->collection` isn't handled here because you already do so in the base Collection\FunctionalTestCase class.